tailwind.config.js

```diff
@@ -81,6 +81,21 @@ module.exports = {
       maxWidth: {
         readable: '60ch',
       },
+      animation: {
+        'slide-in-from-top':
+          'slide-in-from-top 0.3s cubic-bezier(.41,.73,.51,1.02)',
+        leave: 'leave 150ms ease-in forwards',
```

Comment:
`leave` is very generic, we should probably rename it but I can't think of a better name right now.

GPT suggests `fadeOutShrink`, but I'm not a fan.

Reply:
Also, maybe it's not a big deal, but the `leave` animation isn't working. The notification disappears all at once.